src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1442,7 +1458,7 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     status = uct_ud_mlx5_iface_common_init(&self->super.super.super,
                                            &self->ud_common, &config->mlx5_ud);
     if (status != UCS_OK) {
-        goto err_destroy_dct;
+        goto err_destroy_fc_ep_and_dcis;
```

Comment:
can move uct_dc_mlx5_iface_init_fc_ep after uct_ud_mlx5_iface_common_init, to remove uct_dc_mlx5_iface_cleanup_fc_ep(self) from err flow

Reply:
done